src/AudioUnit/AUEnums.cs

```diff
@@ -212,7 +215,9 @@ public enum AudioObjectPropertyScope : uint
 
 	public enum AudioObjectPropertyElement : uint
 	{
+		[Obsolete ("Use the 'Main' element instead.")]
```

Comment:
you can wrap this inside a

```
#if !XAMCORE_4_0
...
#endif
```

so a new, future profile won't require extra work to remove obsolete code.